### DISCUSSION TITLE
Problem with getting student pack

### DISCUSSION THREAD

**Discussion by murtazox04** [community/community#57531] [GitHub Education]
### Hi everyone,

Hi! I'm Murtoza Xurramov. I'm a student of the Karshi branch of Tashkent University of Information Technologies. I tried to get the github student pack but unfortunately I can't. I'm getting a bunch of issues with my github profile not having my full name and my current location being too far from the university. Now we have summer vacation, that's why I came back to my house. And my Github profile name is already correct. And again, my student certificate is in Uzbek and there is no copy of it in English. Please help me!

**Comment by samus-aran** [comment]
Hi @murtazoxurramov 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this is the section that would be key.
